source/blender/blenloader/intern/readfile.c

```diff
@@ -1126,25 +1130,37 @@ static FileData *blo_decode_and_check(FileData *fd, ReportList *reports)
 /* on each new library added, it now checks for the current FileData and expands relativeness */
 FileData *blo_openblenderfile(const char *filepath, ReportList *reports)
 {
-	gzFile gzfile;
-	errno = 0;
-	gzfile = BLI_gzopen(filepath, "rb");
-	
-	if (gzfile == (gzFile)Z_NULL) {
-		BKE_reportf(reports, RPT_WARNING, "Unable to open '%s': %s",
-		            filepath, errno ? strerror(errno) : TIP_("unknown error reading file"));
-		return NULL;
+#ifdef WITH_GAMEENGINE_BPPLAYER
+	int typeencryption = SPINDLE_CheckEncryptionFromFile(filepath);
```

Comment:
own note for after: investigate if it is necessary to add SPINDLE_SetFilePath(filepath) here. I think so.